src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -225,11 +210,8 @@ public async Task DownloadQualityProfile_Success()
             var testSubject = this.CreateTestSubject(bindingArgs, configProviderMock.Object);
 
             host.SupportedPluginLanguages.Add(language);
-            var project = Mock.Of<Project>();
-            testSubject.InternalState.BindingProjects.Add(project);
```

Comment:
For the tests to work with the old code, we had to set both `host.SupportedPluginLanguages` and also configure the `projectToLanguageMapper` to return the languages.

To fix the existing tests I just had to deleted the setup for `IProjectToLanguageMapper` since it is no longer called.